core/src/main/scalajvm/sttp/tapir/FileExtensions.scala

```diff
@@ -0,0 +1,6 @@
+package sttp.tapir
+
+object FileExtensions {
```

Comment:
to consider: maybe this could be part of `TapirExtensions`, meaning that it would be automatically imported in the package object?

+ we would need to extend `TapirAliases` with `trait FileExtensions` as well possible